frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -273,3 +273,22 @@ Feature: Topology chart area
               And user sees shortcut for Access create connector handle
               And user sees shortcut for Qpen quick search modal
               And user sees shortcut for Drag and drop a JAR file into Topology
+
+
+        @regression @manual @odc-5722
+        Scenario: Display of External Bindable resources: T-06-TC24
+            Given user has installed Service Binding operator
+              And user has created external bindable resource
+             When user goes to Topology chart view
+             Then user will see the bindable resource in trapezoid shape
+
+
+        @regression @manual @odc-5722
+        Scenario: Connect to External Bindable resources: T-06-TC25
+            Given user has installed Service Binding operator
+              And user has created external bindable resource
```

Comment:
Provide the data which is used in above scenario

Reply:
Updated in the statement